packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -525,6 +533,42 @@ export class RemoteMediaManager extends EventsScope {
     this.mediaRequestManagers.video.commit();
   }
 
+  /**
+   * Sets which named media group need receiving
+   *
+   *
+   *
+   */
+  public async setReceiveNamedMediaGroup(mediaType: MediaType, languageId: number) {
+    const type = mediaType === MediaType.AudioMain ? 1 : 0;
+    const value = languageId;
+    if (
+      type === this.config.receiveNamedMediaGroup.type &&
+      value === this.config.receiveNamedMediaGroup.value
+    ) {
+      return;
+    }
+
+    this.config.receiveNamedMediaGroup = {
+      type,
+      value,
+    };
+    if (mediaType === MediaType.AudioMain) {
+      this.invalidateCurrentRemoteMedia({
+        audio: true,
+        video: false,
+        screenShareAudio: false,
+        screenShareVideo: false,
+        commit: false,
+      });
+      this.slots.audio.reverse();
```

Comment:
might be worth adding a comment why we're reversing the array here. I guess we're doing it so that we release the slots in reverse order and maintain the same slots in same order after the call to createAudioMedia() is done as they were before setReceiveNamedMediaGroup() was called, right?

Reply:
Right, homer team suggests that we keep same slots for main audio request to get better performance. However I don't think we can always make sure that, e.g. if in future there are more than one named media group request in one media request